index.d.ts

```diff
@@ -875,6 +877,8 @@ declare module 'egg' {
     redirect(url: string, alt?: string): void;
   }
 
+  export interface IContextLocals {}
```

Comment:
Need to extend `PlainObject`, or it will throw error while using the property which was not exist in `locals`.